packages/lisk-p2p/src/peer_discovery.ts

```diff
@@ -18,6 +18,7 @@ import { constructPeerIdFromPeerInfo, Peer } from './peer';
 export interface FilterPeerOptions {
 	readonly blacklist: ReadonlyArray<string>;
 }
+
 // TODO later: Implement LIPS to handle fixed and white list
 export const discoverPeers = async (
```

Comment:
We need to completely stop using `peer.fetchPeers()` for discovery or else we have two different discovery protocols. We should always use the new single-socket discovery approach.

It might seem more efficient to reuse the existing socket on the peer if it exists, but it's going to be a very difficult to manage these two different discovery approaches.

Reply:
Changed as per discussion 👍